### PR TITLE
Changelog 24.5.0

### PR DESCRIPTION
**Pull request by kenodegard**
### Description

✂️ snip snip ✂️ the making of a new release.

Xref #13870

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,78 @@
...
 
+## 24.5.0 (2024-05-08)
+
+### Enhancements
+
+* Report which `MatchSpec` item caused `Invalid*Spec` exceptions for more informative error messages. (#11203 via #13598)
+* MSYS2 packages can now use the upstream installation prefixes. (#13649)
+* Adds a new `reporters` setting for configure output. (#13736)
+* Add support for CEP-15 `base_url` field in `repodata.json`. (#13137 via #13744)
+* In custom channel settings allow specification of channel URLs using a glob-like wildcard pattern, e.g. for user with auth handler plugins. (#13778 via #13779)
```
Comment (beeankha) [inline CHANGELOG.md:11]:
```suggestion
* Add a new `reporters` setting for configure output. (#13736)
* Add support for CEP-15 `base_url` field in `repodata.json`. (#13137 via #13744)
* In custom channel settings, allow specification of channel URLs using a glob-like wildcard pattern, e.g. for user with auth handler plugins. (#13778 via #13779)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,78 @@
...
+
+### Bug fixes
+
+* Fix `conda notices --json` to correctly output JSON. (#13561)
+* Fix prefix replacement for Windows subdir on unixes. (#13689)
```
Comment (beeankha) [inline CHANGELOG.md:16]:
```suggestion
* Fix prefix replacement for Windows subdirectories on Unix systems. (#13689)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,78 @@
...
+* Mark `conda.plan.display_actions` as pending deprecation. (#12421)
+* Mark `conda.plan.execute_actions` as pending deprecation. (#12421)
+* Mark `conda.plan.get_blank_actions` as pending deprecation. (#12421)
+* Mark `conda.plan.install_actions` as pending deprecation. (#12421)
+* Mark `conda.plan.print_dists` as pending deprecation. (#12421)
+* Mark `conda.plan.revert_actions` as pending deprecation. Use `conda.cli.install.revert_actions` instead. (#12421)
+* Mark `conda.plan` as an entrypoint as pending deprecation. (#12421)
+* Mark `conda.activate._Activator.add_export_unset_vars` as pending deprecation. Use `conda.activate._Activator.get_export_unset_vars` instead. (#13720)
+* Mark `conda.activate._Activator.get_scripts_export_unset_vars` as pending deprecation. Use `get_scripts_export_unset_vars` helper function in `test_activate.py` instead. (#13720)
+* Mark `conda.activate._Activator._get_path_dirs(extra_library_bin)` as pending deprecation. Unused argument. (#13720)
```
Comment (beeankha) [inline CHANGELOG.md:35]:
```suggestion
* Mark `conda.activate._Activator._get_path_dirs(extra_library_bin)` as pending deprecation. (#13720)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,78 @@
...
+* Mark `conda install --mkdir` as pending deprecation. Use `conda create` instead. (#13751)
+* Mark `conda._vendor.frozendict` as pending deprecation. Use `frozendict` instead. (#13767 via #13766)
+* Mark `conda.auxlib.collection.make_immutable` as pending deprecation. Use `frozendict.deepfreeze` instead. (#13801)
+* Mark `conda.plan.execute_plan` as pending deprecation. (#13869)
+* Mark `conda.plan.execute_instructions` as pending deprecation. (#13869)
+* Mark `conda.plan._update_old_plan` as pending deprecation. (#13869)
+
+### Docs
+
+* Added type hints and doc strings to `conda.core.index`. (#13816)
```
Comment (beeankha) [inline CHANGELOG.md:48]:
```suggestion
* Add type hints and doc strings to `conda.core.index`. (#13816)
```

File: CHANGELOG.md (hunk 2 above)
Comment (kenodegard) [reply CHANGELOG.md:16]:
```suggestion
* Fix prefix replacement for Windows `subdir` on Unix. (#13689)
```

File: .authors.yml [hunk 5]
```diff
@@ -127,7 +127,7 @@
   first_commit: 2020-06-22 13:01:33
 - name: Daniel Bast
   email: 2790401+dbast@users.noreply.github.com
-  num_commits: 18
```
Comment (tralkasr97) [inline .authors.yml:130]: num_commits: 19

Comment (tralkasr97) [reply .authors.yml:130]: num_commits: 19

**Review by beeankha — APPROVED** [review]

**Pull request merged**